### PR TITLE
2023-3.0 py310 env

### PR DESCRIPTION
**Pull request by mrakitin**
A companion to https://github.com/nsls2-conda-envs/nsls2-collection/pull/23.

```diff
$ diff nsls2-collection/envs/env-py310.yml nsls2-collection-tiled/envs/env-py310.yml
1c1
< name: 2023-3.0-py310
---
> name: 2023-3.0-py310-tiled
33,34c33
<   - databroker >=1.2.5,<2.0.0a
<   - databroker-pack
---
>   - databroker >=2.0.0b29
56d54
<   - intake <=0.6.4
140a139
>   - tiled >=0.1.0a106
```

### CODE REVIEW

**Comment by danielballan** [comment]
The tiled version is correct. We will need newer suitcase (0.4.0, just released on PyPI) and the next datbroker once https://github.com/bluesky/databroker/pull/785 is merged (not yet tagged).

**Comment by danielballan** [comment]
Databroker v2.0.0b30 is released on GitHub and PyPI.

**Comment by danielballan** [comment]
So, we need conda packages for suitcase 0.4.0 and databroker 2.0.0b30, and then we are good.

File: envs/env-py310.yml [hunk 1]
```diff
@@ -107,17 +109,17 @@ dependencies:
   - pymcr
   - pymongo >=3.7
   - pypdf2
-  - pyqt >=5.12.0
+  - pyqt >=5.15.0
   - pyqtgraph
   - pystackreg
   - python-graphviz
   - pyxrf >=1.0.23
```
Comment (dmgav) [inline envs/env-py310.yml:116]:
```suggestion
  - pyxrf >=1.0.24
```

**Review by dmgav — APPROVED** [review]

**Pull request merged**